pkg/morph/client/notary.go

```diff
@@ -430,11 +438,16 @@ func invocationParams(args ...interface{}) ([]sc.Parameter, error) {
 }
 
 // mn returns M and N multi signature numbers. For NeoFS N is a length of
-// inner ring list, and M is a 2/3+1 of it (like in dBFT).
-func mn(ir []*keys.PublicKey) (m int, n int) {
+// inner ring list, and M is a 2/3+1 of it (like in dBFT). If committee is
+// true, returns M as N/2+1.
+func mn(ir []*keys.PublicKey, committee bool) (m int, n int) {
 	n = len(ir)
 	m = n*2/3 + 1
 
+	if committee {
```

Comment:
Lets do
```
if commitee {
  m = n*2/3 + 1
} else {
  m = n/2 + 1
}
```
to underline the mutual exclusive cases.

Reply:
Fixed.